runtime/src/main/java/org/corfudb/runtime/GarbageInformer.java

```diff
@@ -103,6 +124,11 @@ public void stop() {
      * @param locators      A list of locators whose associated SMRRecords are marked as garbage.
      */
     public void addUnsafe(long markerAddress, List<SMRRecordLocator> locators) {
+        // If the function of marking garbage is disable, no garbage decisions are made.
+        if (!rt.getParameters().isGarbageCollectionEnable()) {
```

Comment:
I think you should disable garbage collection in VLO, to not call these APIs at all.

Reply:
moved to VLO.